src/wp-includes/block-patterns.php

```diff
@@ -407,40 +407,28 @@ function _register_theme_block_patterns() {
  */
 
 function _wp_get_block_patterns( WP_Theme $theme ) {
```

Comment:
Unnecessary blank line here, the doc should be immediately before the function.
```suggestion
function _wp_get_block_patterns( WP_Theme $theme ) {
```